### PR TITLE
Gtm8694 test failure fixes

### PR DESCRIPTION
**Pull request by VinayYottaDB**
Moves output of zedit tests to a file and checks for a RESTRICTEDOP error to standardize the test output

### CODE REVIEW

File: v63003/inref/gtm8680.m [hunk 1]
```diff
@@ -17,26 +17,22 @@
...
 	set max=100000
 	set maxit=0
-	set maxit2=0
 	set minit=0
-	set minit2=0
-	set ^its($increment(^i))=0
+	set its($increment(it))=0
 	set start=$h,cur=start,prev=start,previ=0,t=$$^difftime(cur,start)
-	for i=1:1 do  quit:(t>15)
+	for i=1:1 do  quit:(t>30)
```
Comment (nars1) [inline v63003/inref/gtm8680.m]: Why does this change show up in this pull request? All following changes fall in the same category. Some mixup.

File: v63002/inref/gtm8694.m [hunk 2]
```diff
@@ -60,7 +60,8 @@
 zeditfn
 	write "# TESTING ZEDIT",!
 	set $etrap="do incrtrap^incrtrap"
-	zedit "dummy.txt"
+	set dir=$ZDIRECTORY
+	zedit dir_"/dummy.txt"
```
Comment (nars1) [inline v63002/inref/gtm8694.m]: Why was this full-path change required?

File: v63002/u_inref/gtm8694.csh [hunk 3]
```diff
@@ -126,15 +126,21 @@ TRIGGER_MOD:$groupname
...
 $ydb_dist/mumps -run breakfn^gtm8694 >& unrestrictedBREAK.txt
 cat unrestrictedBREAK.txt
 $ydb_dist/mumps -run zbreakfn^gtm8694 >& unrestrictedZBREAK.txt
 cat unrestrictedZBREAK.txt
 $ydb_dist/mumps -run zcmdlnefn^gtm8694 "ZCMDLNE was not ignored" >& unrestrictedZCMDLINE.txt
 cat unrestrictedZCMDLINE.txt
+echo "# TESTING ZEDIT"
 $ydb_dist/mumps -run zeditfn^gtm8694 >& unrestrictedZEDIT.txt
-cat unrestrictedZEDIT.txt
+if ("" == `$grep RESTRICTEDOP unrestrictedZEDIT.txt`) then
```
Comment (nars1) [inline v63002/u_inref/gtm8694.csh]: As discussed, please include a comment as to WHY this change was needed. Either here in the script or in the commit message detail.

File: v63002/inref/gtm8694.m (hunk 2 above)
Comment (VinayYottaDB) [reply v63002/inref/gtm8694.m]: When i recieved the error regarding it not finding the file i included this to try to circumvent it. It did nothing, so I can delete it
